### PR TITLE
Grept apply 1709887482

### PR DESCRIPTION
**Pull request by herms14**

### CODE REVIEW

**Comment by herms14** [comment]
@Azure/avm-core-team for your review and approval

File: variables.tf [hunk 1]
```diff
@@ -122,70 +145,50 @@ variable "tracing_tags_prefix" {
...
+variable "virtual_network_dns_servers" {
+  type = object({
+    dns_servers = list(string)
+  })
+  default     = null
+  description = "(Optional) List of IP addresses of DNS servers"
+}
 
-  }
+variable "vnet_location" {
```
Comment (matt-FFFFFF) [inline variables.tf:167]: variables pertaining the the primary resource, in this case the virtual network, should not be prefixed

File: variables.tf [hunk 2]
```diff
@@ -122,70 +145,50 @@ variable "tracing_tags_prefix" {
...
-    marketplace_partner_resource_id          = optional(string, null)
-  }))
-  default  = {}
-  nullable = false
-
-  validation {
-    condition     = alltrue([for _, v in var.diagnostic_settings : contains(["Dedicated", "AzureDiagnostics"], v.log_analytics_destination_type)])
-    error_message = "Log analytics destination type must be one of: 'Dedicated', 'AzureDiagnostics'."
-  }
+variable "vnet_name" {
```
Comment (matt-FFFFFF) [inline variables.tf:175]: See above shoudl call this `name`

**Review by matt-FFFFFF — CHANGES_REQUESTED** [review]
Hey! this is fantastic work, couple of comments on variable naming

**Comment by herms14** [comment]
Thanks for the review @matt-FFFFFF. Names has been updated now.

**Review by matt-FFFFFF — APPROVED** [review]

**Pull request merged**